conans/client/rest/rest_client.py

```diff
@@ -144,7 +144,7 @@ def get_package_info(self, package_reference):
         contents = {key: decode_text(value) for key, value in dict(contents).items()}
         return ConanInfo.loads(contents[CONANINFO])
 
-    def get_recipe(self, conan_reference, dest_folder):
+    def get_recipe(self, conan_reference, dest_folder, retrieve_sources=False):
```

Comment:
The method already checks CONANFILE existence within, so it is not a big deal IMHO.
Just help me define the interface. A single parameter ``skip_files`` is not enough, because actually there are two opposite cases:
- Retrieve all files except EXPORT_SOURCES_TGZ_NAME
- Retrieve only EXPORT_SOURCES_TGZ_NAME
So, a general paremeterization would be:
- You want to retrieve all files but those included in ``skip_files``
- You want to retrieve only the files defined by? Another parameter ``only_these_files``?
Then you have possible inconsistent state, like having the same file both in ``skip_files`` and ``only_these_files``.
Another possibility is two parameters:
- One ``files`` list
- One tri-state (None/False, True, exclude/include the files in the list).
Still possible inconsistent state like exclude!=None with empty ``files`` list.
IMO, too much issues when we exactly know what file is to be managed differently. But please suggest the interface you'd like.